tests/test_current_jumps.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from src.current_jumps import std_1D
+from src.current_jumps import get_n_moms_of_moving_array
+import pytest
+
+# list of inputs and expected results for std_1D() function
+def list_to_test_std_1D():
```

Comment:
these functions are not test, hence they should go into a subdirectory called test_utils, and imported from there.